Tools/CISupport/ews-build/steps.py

```diff
@@ -2829,6 +2833,7 @@ def filter_logs_containing_error(self, logs, searchString='rror:', max_num_lines
                 filtered_logs.append(line)
         return '\n'.join(filtered_logs[-max_num_lines:])
 
+    @defer.inlineCallbacks
```

Comment:
since `send_email_for_new_build_failure` now has `@defer.inlineCallbacks`, should it's invocation inside AnalyzeCompileWebKitResults.analyzeResults() also have `yield`? (or is that not necessary?)

Reply:
Pedantically, yes. And I intend to do that in the coming days.

Practically, though, no. `send_email_for_new_build_failure` is basically operating as a sub-routine instead of a function. Since its caller doesn't care about the result, its caller doesn't need to wait for the result. Now, this means that `send_email_for_new_build_failure` can actually finish _after_ it's caller does, which is a bit weird and not great, but it shouldn't break anything.

The reason I didn't include modifications to `AnalyzeCompileWebKitResults.analyzeResults` in this change is because those changes are higher risk, and I was trying to fix the failure to apply the `merging-blocked` label as quickly as possible.

Reply:
ok, sounds good.